presto-hive/src/test/java/io/prestosql/plugin/hive/HiveQueryRunner.java

```diff
@@ -126,6 +128,12 @@ public static DistributedQueryRunner createQueryRunner(Iterable<TpchTable<?>> ta
         }
     }
 
+    private static void setupLogging()
+    {
+        Logging logging = Logging.initialize();
+        logging.setLevel("org.apache.parquet.hadoop", WARN);
```

Comment:
I would setup this to `org.apache`, then I would move it to `DistributedQueryRunner`

Reply:
There could be useful INFO `org.apache` logs though